src/voting-contracts/migrations/5_strategies_migration.js

```diff
@@ -0,0 +1,8 @@
+const Strategies = artifacts.require("Strategies");
+const {BN} = require('bn.js');
+
+const ERC20 = "0x6483834D130ca21F934153cB40ee12136A56b4f2";
```

Comment:
these addresses should not be fixed in migration files, as they can be different for each instance. E.g. `truffle networks` returned this for my current instance 
```
 % truffle networks 
-- snip --
Network: UNKNOWN (id: 5777)
  CorruptERC20: 0x74c0E0Df2de42691D7C8dFAfD46F5386Da327d4C
  Migrations: 0xF529DdBa7D70D21503F9cBC8224a9F53850e1702
  Proposal: 0xB519Ad89ccC8cC782944f0D98763D5aAd7fF074F
  ProposalFactory: 0xE1e7aAEa0203DF2DaFf38d1fFF1a3846b2e177B0
  SampleERC20: 0xa5F8f888CCc00685a73E9F68DEe5971305755d7E
  Strategies: 0xB1DBD1E43D52982E2B9428B6B94F864b54980B1e
```
check `2_proposal_factory_migrations.sol` for further details

Reply:
Thanks for pointing out. I have updated the migration files.